src/client/components/Avatar/index.js

```diff
@@ -0,0 +1,39 @@
+/* @flow */
+
+import React from 'react'
+import { default as MaterialAvatar } from 'material-ui/Avatar'
+
+export type Props = {
+  style?: Object,
+  avatar: string,
+};
+
+const avatarStyle = {
+  borderRadius: '20%',
+  float: 'left',
+  display: 'table-column',
+  height: '40px',
+  width: '40px',
+  color: 'rgba(157, 156, 156, 0.58)',
+}
```

Comment:
Is there a compelling reason for not placing it in a css and using a className for it instead?